ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/OffsetBuilderSpec.scala

```diff
@@ -14,67 +14,68 @@ class OffsetBuilderSpec extends AnyWordSpec with Matchers with ScalaCheckDrivenP
 
   "OffsetBuilder" should {
```

Comment:
I think it's important to make sure we test corner cases around the 7-bytes `Long`.